templates/hp_comware_display_lldp_neighbor-information_verbose.textfsm

```diff
@@ -0,0 +1,17 @@
+Value Required LOCAL_IF (\S+)
```

Comment:
Can you please update the value names to match other LLDP templates (e.g. cisco_ios_show_lldp_neighbors_detail.textfsm)

The **CHASSIS_ID** may be better as `\S+` as well.

Reply:
Yes,I have modified it and added some other changes based on cisco_ios_show_lldp_neighbors_detail.textfsm.